frontend/src/scenes/data-warehouse/ViewLinkModal.tsx

```diff
@@ -35,8 +35,9 @@ interface ViewLinkFormProps {
 }
 
 export function ViewLinkForm({ tableSelectable }: ViewLinkFormProps): JSX.Element {
-    const { viewOptions, toJoinKeyOptions, selectedView, selectedTable, fromJoinKeyOptions } = useValues(viewLinkLogic)
-    const { selectView, toggleFieldModal, selectTable } = useActions(viewLinkLogic)
+    const { viewOptions, toJoinKeyOptions, selectedView, selectedTableName, fromJoinKeyOptions } =
+        useValues(viewLinkLogic)
+    const { selectView, toggleFieldModal, selectTableName } = useActions(viewLinkLogic)
     const { tableOptions } = useValues(databaseSceneLogic)
```

Comment:
NIT: can import this from viewLinkLogic now!

Reply:
Perfect, updated!